src/utils/browser-storage.js

```diff
@@ -0,0 +1,30 @@
+import { isBrowser } from './is-browser';
```

Comment:
Basically identical to https://github.com/mongodb/snooty/blob/a37358d314fc0468acb5bf82fc4f8919279ddb76/src/utils/browser-storage.js